.go-version

```diff
@@ -0,0 +1 @@
+1.16.5
```

Comment:
Is this version used for building the binary itself? Nomad has to be built with go 1.17+.

Reply:
Yes, `.go-version` is where the go-version is set and used for all CI build jobs. I just updated this to 1.17.5. Thanks for the note!

Reply:
Just wanted to confirm this is needed for this older release branch. go mod shows Go 1.16, and the builds succeed with Go 1.16.5 but not 1.17.5. I can dig into this more next week, but just want to clarify that this branch was previously building with 1.17.5 before spending the time on it.

Reply:
Oh shoot, I'm sorry. I wasn't paying close enough attention and didn't realize this was a separate PR for 1.0.x. You're right, that's on 1.16 still. 🤦

Reply:
No worries! I'll revert back to 1.16 for now.